### PR TITLE
Huda3

### PR DESCRIPTION
**Pull request by kpslp**
made changes and added invite users

@SenthilKumarSivakami

### CODE REVIEW

File: invite-users.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
 # Invite users
 
-Public officers and vendors need to have a [TechPass](https://www.developer.tech.gov.sg/products/categories/digital-identity/techpass/overview.html) account to access [Singapore Government Tech Stack(SGTS)](https://www.developer.tech.gov.sg/singapore-government-tech-stack/overview/index.html) products. TechBiz allows public officers to send TechPass invites to other public officers or create a TechPass account on behalf of public officers and vendors.
+Public officers and vendors are required to have a [TechPass](https://www.developer.tech.gov.sg/products/categories/digital-identity/techpass/overview.html) account to access [Singapore Government Tech Stack(SGTS)](https://www.developer.tech.gov.sg/singapore-government-tech-stack/overview/index.html) products. TechBiz enables public officers to send TechPass invites to other public officers or create a TechPass account on behalf of other public officers and vendors.
```
Comment (SenthilKumarSivakami) [inline invite-users.md:3]: retain only "other" all other changes pls revert.

File: invite-users.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 
-## Prerequisites
+## Pre-requisites
 
-- Only a public officer with a valid organisation email address can invite another public officer or vendor. For example, *your_name@agency.gov.sg* using a [non-SE GSIB](glossary) device.
+- Public officers with a valid organisation email address such as  *your_name@agency.gov.sg* can invite another officer or vendor using a [non-SE GSIB](glossary) device.
```
Comment (SenthilKumarSivakami) [inline invite-users.md:7]: other public officers or vendors....

File: invite-users.md [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
 
-- Have the following information ready before proceeding:
+- Have the following information ready:
```
Comment (SenthilKumarSivakami) [inline invite-users.md:9]: let's keep the previous sentence.

File: invite-users.md [hunk 4]
```diff
@@ -19,9 +19,9 @@ Public officers and vendors need to have a [TechPass](https://www.developer.tech
 
 1.  [Log in to TechBiz portal.](log-in-to-TechBiz-portal.md)
 
-2.  In **Overview**, click **Invite users**. Alternatively, from the left sidebar, click **Invite Users**.
+2.  In **Overview**, go to **Invite users**. Alternatively, from the left sidebar, go to **Invite Users**.
```
Comment (SenthilKumarSivakami) [inline invite-users.md:22]: select

File: invite-users.md [hunk 5]
```diff
@@ -19,9 +19,9 @@ Public officers and vendors need to have a [TechPass](https://www.developer.tech
...
 
-3.  Select the required **invitation type**, enter the required details and click **Submit**.
+3.  Select the required **invitation type**, enter the required details and  **Submit**.
```
Comment (SenthilKumarSivakami) [inline invite-users.md:24]: pls revert to the previous.

File: invite-users.md [hunk 6]
```diff
@@ -91,3 +91,16 @@ A success message with the user list is displayed.
 > - If you are a public officer, click [Onboard to SEED](https://docs.developer.tech.gov.sg/docs/techpass-user-guide/#/onboard-to-seed) and follow the on-screen instructions.
 > - If you are a vendor, raise a [service request](https://go.gov.sg/techpass-sr) in the TechPass service desk.
 
+To view list of users invited:
```
Comment (SenthilKumarSivakami) [inline invite-users.md:94]: To view the list of invited users:

File: invite-users.md [hunk 7]
```diff
@@ -91,3 +91,16 @@ A success message with the user list is displayed.
...
+
+1.  [Log in to TechBiz portal.](log-in-to-TechBiz-portal.md)
+
+2.  In **Overview**, click **Invite users**. Alternatively, from the left sidebar, go to **Invite Users**.
+
+3. Go to **Invite user status**
```
Comment (SenthilKumarSivakami) [inline invite-users.md:100]: period.

File: invite-users.md [hunk 8]
```diff
@@ -91,3 +91,16 @@ A success message with the user list is displayed.
 > - If you are a public officer, click [Onboard to SEED](https://docs.developer.tech.gov.sg/docs/techpass-user-guide/#/onboard-to-seed) and follow the on-screen instructions.
 > - If you are a vendor, raise a [service request](https://go.gov.sg/techpass-sr) in the TechPass service desk.
 
+To view list of users invited:
+
+1.  [Log in to TechBiz portal.](log-in-to-TechBiz-portal.md)
+
+2.  In **Overview**, click **Invite users**. Alternatively, from the left sidebar, go to **Invite Users**.
+
```
Comment (SenthilKumarSivakami) [inline invite-users.md:99]: remove extra spaces.

File: invite-users.md [hunk 9]
```diff
@@ -91,3 +91,16 @@ A success message with the user list is displayed.
...
+3. Go to **Invite user status**
+
+The list of users invited and their status will be shown. Their TechPass and SEED status will be reflected on the list.
```
Comment (SenthilKumarSivakami) [inline invite-users.md:102]: what status of TechPass and SEED is displayed and how should the user this list.

File: invite-users.md (hunk 6 above)
Comment (SenthilKumarSivakami) [reply invite-users.md:94]: Specify the context when a user should select this option?

File: invite-users.md [hunk 10]
```diff
@@ -91,3 +91,16 @@ A success message with the user list is displayed.
...
+
+<img src="/images/invite_user_status.png" alt="drawing" width="55%"/>
+
+First name and last name is reflected when user has signed up for TechPass.
```
Comment (SenthilKumarSivakami) [inline invite-users.md:106]: need to elaborate the different usecases.

File: notifications.md [hunk 11]
```diff
@@ -4,11 +4,13 @@ TechBiz notifies users of their request status and tasks awaiting their action.
...
 
-2. Click <img src="images/bell.png" alt="drawing" width="5%"/> 
+2. Click <img src="images/bell_1.png" alt="drawing" width="5%"/> 
 
 
 3. Click the required pending task.
-<img src="images/notification_1.png" alt="drawing" width="50%"/> 
+<img src="images/notifnew.png" alt="drawing" width="55%"/> 
+
+Red dot refers to unread notifications
```
Comment (SenthilKumarSivakami) [inline notifications.md:13]: please add this as a note.

File: invite-users.md (hunk 6 above)
Comment (kpslp) [reply invite-users.md:94]: done

File: invite-users.md (hunk 8 above)
Comment (kpslp) [reply invite-users.md:99]: done

File: invite-users.md (hunk 9 above)
Comment (kpslp) [reply invite-users.md:102]: done

File: invite-users.md (hunk 10 above)
Comment (kpslp) [reply invite-users.md:106]: done

**Pull request merged**